_data/api-swagger.yaml

```diff
@@ -1053,6 +1054,28 @@ definitions:
       For example, in the Netherlands, personal number (BSN) can not be used for privacy reasons
       and is substituted by document number such as passport number, but in other countries this poses no problem.
 
+  OpenID Providers:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      description: |
+        Adds an OpenID verification step to the signing process.
+      properties:
+        ProviderName:
+          type: string
+          description: |
+            (OPTIONAL) When value is provided, it should match the configured one for the organization.
```

Comment:
explicitly refer to "the configured one"

Reply:
Provider Names is configured in a free flexible string way, we do not have hard coded names. So what do you refer here to be explicit?